new/LIT7097SanaSom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7097SanaSom" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸንአ፡ ጾም፡ ነግረ፡ ኢሳይያ፡ አብ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣanʾa ṣom nagra ʾIsāyǝyā ʾab (Poetic composition)</title>
+                <title xml:lang="en" corresp="#t1">The fast grew strong, said father ʾIsāyǝyā…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                    <p>Poetic composition, defined in the manuscript as a <foreign xml:lang="gez">Gubāʾa Qānā</foreign>.
+                        However, no rhymes can be discerned, as one would expect in a <foreign xml:lang="gez">Gubāʾa Qānā</foreign>-type
+                        <foreign xml:lang="gez">qəne</foreign>. 
+                        The composition has no author attribution in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
```

Comment:
You think it is not justified to add Qene at least as keyword here?

Reply:
I'm not sure. What do you say, @DenisNosnitsin1970 and @CarstenHoffmannMarburg ?

Reply:
> It has an explicit qene-type ("Guba'e Qana"), but I am not able to discern any rhyme. If you note something, please let me know.

It has a crossed rhyme ending in ብ abd ሉ/ልው. Important for the definition of Guba'e Qana is the rhyme in አሉ/እልው, because Guba'e Qana has only two verses according to Guidi.
I think, it is legitimate to add the keywords Qene and Guba'e Qana and note the extraordinary rhyme.

Reply:
Are you suggesting that this is in fact two different two-line qenes?

Reply:
Based on what Denis commented, I would analyse what we have in the Uppsala manuscript as _one_ text, because it follows the layout for individual texts in the manuscript and has one number in the numbering of the manuscript, etc. But this text is not strictly a qene, because it breaks the rules. Instead, I would say that it is composed of two qenes, one of which is attested elsewhere, as noted by Denis, and therefore should have its own LIT ID (= LIT7102). This is what I tried to convey in the abstract :)